libs/net/entities/Content.cs

```diff
@@ -124,6 +124,7 @@ public class Content : AuditColumns
     /// get/set - The page this story was found one.
     /// </summary>
     [Column("page")]
+    [MaxLength(10, ErrorMessage = $"{nameof(Page)} only allow up to 10 characters.")]
```

Comment:
We don't need this.  We already control the constraint in the `ContentConfiguration.cs` class.  Having this on each model will just make maintaining this more difficult and will cause regression issues.

What are you hoping to achieve with this?

Reply:
Removed as the constraint is already set somewhere else as mentioned, initially thought to be more explicit but yes there would be no need for it, thanks